client/packages/common/src/hooks/useSimplifiedTabletUI/useSimplifiedTabletUI.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * This hook determines whether or not the UI should be displayed with the
+ * "simplified" options when on a mobile device.
+ *
+ * Use the value returned from this hook whenever a UI element is conditional on
+ * the "simplified" mobile view.
+ *
+ * Three criteria must be true:
+ * - The store pref "Use Simplified Mobile UI" is enabled
+ * - The (old) preference "Pack size to 1" is enabled
+ * - The device screen size is "tablet" size or smaller
+ */
+
+import { useAppTheme, useMediaQuery, Breakpoints } from '@common/styles';
+import { useAuthContext, usePreference } from '../../authentication';
+import { PreferenceKey } from '@common/types';
+
+export const useSimplifiedTabletUI = () => {
+  const theme = useAppTheme();
+  const isMediumScreen = useMediaQuery(theme.breakpoints.down(Breakpoints.lg));
+
+  const { data } = usePreference(PreferenceKey.UseSimplifiedMobileUi);
+
+  const { store } = useAuthContext();
+
+  // Defaulting these values to `true` -- if the value hasn't loaded yet, it's
+  // better that the UI shows too simplified than too crowded
```

Comment:
Is this reasonable? Or will it not really matter (i.e. it won't render before these values are loaded)?

Reply:
I've got no idea on the circumstances of rendering before loading values - but seems costless and agree good to render more simply as a default